### ISSUE TITLE
[Good First Issue]: Support aten::logit, aten::special_logit

### ISSUE THREAD

**Issue by code-dev05** [openvinotoolkit/openvino#28744]
### Context

**Context**
OpenVINO component responsible for support of PyTorch models is called as PyTorch Frontend (PT FE). PT FE converts a model represented as TorchScript model to a model in [OpenVINO opset](https://docs.openvino.ai/latest/openvino_docs_ops_opset.html).

Add Support for the following operations:
`aten::logit`

### What needs to be done?

Implement conversion rule and/or transformation to support the new operation.
Implement operation tests in tests/layer_tests/pytorch_tests. Please consider different data types, but keep reasonable number of test cases

**Example Pull Requests**
[https://github.com/openvinotoolkit/openvino/pull/18998](https://github.com/openvinotoolkit/openvino/pull/18998)

### Resources

- [Contribution guide - start here!](https://github.com/openvinotoolkit/openvino/blob/master/CONTRIBUTING.md)
- [Intel DevHub Discord channel](https://discord.gg/7pVRxUwdWG) - engage in discussions, ask questions and talk to OpenVINO developers
- [How to link your Pull Request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#manually-linking-a-pull-request-to-an-issue-using-the-pull-request-sidebar)

### Contact points

@openvinotoolkit/openvino-pytorch-frontend-maintainers

_No response_

**Comment by code-dev05** [comment]
.take

**Comment by code-dev05** [comment]
I have been trying to implement this operation and facing difficulty with the decomposition. It would be great if you could help me figure it out.

@mvafin

**Comment by mvafin** [comment]
> I have been trying to implement this operation and facing difficulty with the decomposition. It would be great if you could help me figure it out.
> 
> [@mvafin](https://github.com/mvafin)

@code-dev05  Check the formula here https://pytorch.org/docs/stable/special.html#torch.special.logit
If the eps is not None you need to clamp the input with [Clamp-1](https://docs.openvino.ai/2024/documentation/openvino-ir-format/operation-sets/operation-specs/activation/clamp-1.html), if it is None, then leave as-is.
After that apply the formula for ln(z/(1-z))

**Comment by mvafin** [comment]
Please also support `aten::special_logit` as it seems to be just an alias to the same operation.

**Comment by code-dev05** [comment]
Can we perform the operation `1 - z` by using a constant 1 and doing subtraction with z?

**Comment by p-wysocki** [comment]
Hello @code-dev05, do you have any questions or require any help?

**Comment by code-dev05** [comment]
Yes, I am facing an accuracy issue. The fp16 test cases are passing but fp32 are not. Can you guide me how to go about it?
@p-wysocki

**Comment by p-wysocki** [comment]
@openvinotoolkit/openvino-pytorch-frontend-maintainers @mvafin could you please take a look?

**Comment by code-dev05** [comment]
This is the PR - [28882](https://github.com/openvinotoolkit/openvino/pull/28882)
I opened it some time ago.

**Comment by codewarnab** [comment]
i can work on this is issue if it still unsolved

**Comment by Raakshass** [comment]
.take

**Comment by Raakshass** [comment]
hey just want to confirm that since ive picked this task then why is it still open for contribution needed?

**Comment by SondosAlaaeldin** [comment]
.take
